content/notifications/send-notifications/developer-guides/send-sms-notifications/_index.en.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Send sms notifications 
+linktitle: Send sms notifications 
+description: Endpoint for sending an sms to one or more recipient with known contact details.
+weight: 50
+toc: true
+---
+
+## Endpoint
+
+POST /orders/sms
+
+## Authentication
+
+This API requires authentication and the request must also include one of the following: 
+
+- Maskinporten scope __altinn:serviceowner/notifications.create__ (for external system callers) 
+- Platform Access Token (for Altinn Apps and internal Altinn systems)
+
+See [Authentication and Authorization](../../../api/#authentication--authorization) for more information.
+
+## Request
+
+### Content-type
+
+application/json
+
+
+### Request body
+The request body must contain the order request formatted as an
+[SmsNotificationOrderRequestExt](https://github.com/Altinn/altinn-notifications/blob/main/src/Altinn.Notifications/Models/SmsNotificationOrderRequestExt.cs)
+and serialized as a JSON string.
+
+
+### Required order request properties  
+
+#### __body__
+Type: _string_
+
+The contents of the SMS.
+
+#### recipients
+Type: _List of [RecipientExt](https://github.com/Altinn/altinn-notifications/blob/main/src/Altinn.Notifications/Models/RecipientExt.cs)_
+  
+A list containing one or more recipient objects, each representing a recipient with a mobile number.
+
+### Optional order request properties
+#### __senderNumber__
```

Comment:
Inconsistent use of BOLD in some (forth) titles

Reply:
First property must be bold for formatting purposes. Without the '__ __' the first property is centered for some reason :S